scripts/ocr_runner.py

```
@@ -0,0 +1,198 @@
+        wf.write(rf.read())
+    tmp.unlink()
+
+
+def main(batch, min_pdf_bytes, max_text_bytes, ocr_timeout):
+    manifest = load_manifest()
+    candidates = []
+    for txt in TEXT_DIR.glob('*.txt'):
+        try:
+            size = txt.stat().st_size
+        except Exception:
+            continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)